meta-resin-pyro/recipes-core/systemd/systemd_232.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/${PN}-${PV}"
+
+# This patch is available from 236 so stop applying this patch after poky is
+# updated to a version which provides this version or a later one
+SRC_URI_append = "file://0001-resolved-fix-loop-on-packets-with-pseudo-dns-types.patch"
```

Comment:
This can be made so that it only applies to versions until 236

Reply:
That is a good idea. I pushed an updated PR.